### ISSUE TITLE
Grey color is difficult to see on white background

### ISSUE THREAD

**Issue by sharvil** [tensorflow/tensorboard#1574]
TensorBoard version (from pip package, also printed out when running tensorboard)
1.10.0
TensorFlow version if different from TensorBoard
1.10.0
OS Platform and version (e.g., Linux Ubuntu 16.04)
Ubuntu 16.04
Python version (e.g. 2.7, 3.5)
3.5.2

See image below. Same applies to unsmoothed values in "Scalars" tab. While we're on the subject, darkening older summaries in the histogram seems entirely backwards to me. I'm typically interested in the most recent summaries so the contrast against the background for the newest items should be highest.

I recommend: 1) don't use greyscales for any runs since they easily blend into the background and share a common color with the plot metadata 2) use darker, high-contrast colors for more recent summaries and blend towards background color for older summaries.

<img width="338" alt="screen shot 2018-11-02 at 3 25 20 pm" src="https://user-images.githubusercontent.com/391004/47943497-86fc6400-deb3-11e8-97ca-3b16a8aa271d.png">

**Comment by sharvil** [comment]
I think the submitted CL is unrelated to this bug.

**Comment by stephanwlee** [comment]
@sharvil  you are correct. I have made a mistake and it seems like I can't retroactively fix it :( sorry about that.

**Comment by sharvil** [comment]
Bumping this up. This is still an issue – at the very least, can you just remove the grey color from the set of possible colors Tensorboard uses?

**Comment by sharvil** [comment]
Ping.

**Comment by sharvil** [comment]
Ping.

**Comment by sharvil** [comment]
Is there an update on this issue?

**Comment by stephanwlee** [comment]
You may have noticed some movement in https://github.com/tensorflow/tensorboard/issues/893#issuecomment-726181438.

```
2) use darker, high-contrast colors for more recent summaries and blend towards background color for older summaries.
```

While we can definitely apply, please consider using time series and change the color to your liking as workaround.

**Comment by sharvil** [comment]
Great! Thanks for the update. Looks like time series fixes this issue by removing this grey color from the palette.
